gateway/src/main/java/io/zeebe/gateway/impl/configuration/ConfigurationDefaults.java

```diff
@@ -20,8 +20,9 @@
   public static final String DEFAULT_HOST = "0.0.0.0";
   public static final int DEFAULT_PORT = 26500;
   public static final String DEFAULT_CONTACT_POINT_HOST = "127.0.0.1";
-  public static final int DEFAULT_CONTACT_POINT_PORT = 26501;
+  public static final int DEFAULT_CONTACT_POINT_PORT = 26505;
```

Comment:
Configuration changes have to be adjusted in the distribution configuration file